decidim-verifications/config/locales/en.yml

```
@@ -79,7 +79,10 @@ en:
     events:
       verifications:
         verify_with_managed_user:
-          notification_title: The participant <a href="%{resource_path}">%{resource_title}</a> has tried to verify herself with the data of the managed participant <a href="%{managed_user_path}">%{managed_user_name}</a>
+          email_intro: The participant <a href="%{resource_path}">%{resource_title}</a> has tried to verify herself with the data of the managed participant <a href="%{managed_user_path}">%{managed_user_name}</a>.
+          email_outro: Check the <a href="/admin/conflicts">Verifications's conflicts list</a> and contact the participant to verify her details and solve the issue.
```

```suggestion
          email_outro: Check the <a href="%{conflicts_path}">Verifications's conflicts list</a> and contact the participant to verify their details and solve the issue.
```
Let's use the path instead of hard-coding the URL